cmd/bbolt/command_buckets.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	bolt "go.etcd.io/bbolt"
+)
+
+func newBucketsCommand() *cobra.Command {
+	var bucketsCmd = &cobra.Command{
+		Use:   "buckets <path>",
+		Short: "print a list of buckets",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return bucketsFunc(cmd, args[0])
+		},
+	}
+	return bucketsCmd
+}
+
+func bucketsFunc(cmd *cobra.Command, srcDBPath string) error {
+	// Required database path.
+	if srcDBPath == "" {
+		return ErrPathRequired
```

Comment:
I think the `cobra.ExactArgs(1)` already guarantees that it won't be empty string? Pleasae follow the same pattern as other command.

Reply:
This check is from the original code.

Reply:
"the original code" doesn't mean that it's necessary.

Please also always remember to simplify the the implementation,

https://github.com/etcd-io/bbolt/blob/cb0618b9da2b436b7bed6013b515697cbaeb458a/cmd/bbolt/command_inspect.go#L26-L30